recipes/pom.xml

```diff
@@ -13,7 +13,7 @@
     <name>Quarkus Updates - Recipes</name>
 
     <properties>
-        <maven.compiler.release>11</maven.compiler.release>
+        <maven.compiler.release>17</maven.compiler.release>
```

Comment:
Java 17 is used for this change, because it malkes writing the test much easier because of the String block. I think that it is not a strong enough argument to force java version update on the quarkus-recipes.